lib/events.js

```diff
@@ -436,6 +436,14 @@ function listenerCount(type) {
   return 0;
 }
 
+EventEmitter.prototype.eventNames = function() {
+  const events = this._events;
+  if (events) {
```

Comment:
Maybe it would be better to instead check `this._eventsCount > 0` in case of an empty `_events` object?